frontend/src/pages/modelServing/screens/global/InferenceServiceProject.tsx

```diff
@@ -27,7 +27,22 @@ const InferenceServiceProject: React.FC<InferenceServiceProjectProps> = ({ infer
 
   const project = modelServingProjects.find(byName(inferenceService.metadata.namespace));
 
-  return <>{project ? getProjectDisplayName(project) : 'Unknown'}</>;
+  return (
+    <>
+      {project ? (
+        <>
+          {getProjectDisplayName(project)}{' '}
```

Comment:
We could give a little bit of padding, but that can be a follow up enhancement and we can discuss it later with UX